### ISSUE TITLE
syntax error in gen_images.py?

### ISSUE THREAD

**Issue by duskvirkus** [NVlabs/stylegan3#6]
**Describe the bug**

gen_images.py might have a syntax error. Don't know if this is my config or a bug with the repo.

```
  File "gen_images.py", line 34
    if m := range_re.match(p):
          ^
SyntaxError: invalid syntax
```

**To Reproduce**
1. In '...' directory, run command '...'
2. See error (copy&paste full log, including exceptions and **stacktraces**).

See colab notebook:

https://gist.github.com/duskvirkus/46257dfc709854577128c0a04ffda987

Please copy&paste text instead of screenshots for better searchability.

**Expected behavior**

**Screenshots**

![Screenshot from 2021-10-11 22-35-33](https://user-images.githubusercontent.com/43045568/136892173-b10905de-dda6-4e69-9aef-2c87443b21fc.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Linux Ubuntu 20.04, Windows 10] ubuntu (colab instance)
 - PyTorch version (e.g., pytorch 1.9.0)
 - CUDA toolkit version (e.g., CUDA 11.4)
 - NVIDIA driver version
 - GPU [e.g., Titan V, RTX 3090]
 - Docker: did you use Docker?  If yes, specify docker image URL (e.g., nvcr.io/nvidia/pytorch:21.08-py3)
![Screenshot from 2021-10-11 22-44-38](https://user-images.githubusercontent.com/43045568/136892845-4b3dd235-fa20-4169-9450-4f0e0c67d909.png)

no docker

**Additional context**

**Comment by duskvirkus** [comment]
My fix was to change.

https://github.com/NVlabs/stylegan3/blob/68d83b5167b266c68415629c0bc687c714d0835e/gen_images.py#L34
to 
```python
        m = range_re.match(p)
        if m:
```

**Comment by nurpax** [comment]
Thanks for the bug report!  It's indeed not a bug: you will need Python 3.8 at minimum.  (Python 3.9 should also work, that's what I use).

We upped the minimum spec as we use Python 3.8+ internally and decided not to spend time testing and potentially backporting fixes to Python 3.7.

If you for some reason cannot upgrade past Python 3.7, you can still fairly easily workaround this.  (But it may be harder and harder in future code drops.)
